src/renault_api/kamereon/models.py

```diff
@@ -361,6 +361,53 @@ def for_json(self) -> Dict[str, Any]:
         }
 
 
+@dataclass
+class HvacDaySchedule(BaseModel):
+    """Kamereon vehicle hvac schedule for day."""
+
+    readyAtTime: Optional[str]  # noqa: N815
+
+    def for_json(self) -> Dict[str, Any]:
+        """Create dict for json."""
+        return {
+            "readyAtTime": self.readyAtTime,
+        }
+
+
+@dataclass
+class HvacSchedule(BaseModel):
+    """Kamereon vehicle hvac schedule for week."""
+
+    id: Optional[int]
+    activated: Optional[bool]
+    monday: Optional[HvacDaySchedule]
+    tuesday: Optional[HvacDaySchedule]
+    wednesday: Optional[HvacDaySchedule]
+    thursday: Optional[HvacDaySchedule]
+    friday: Optional[HvacDaySchedule]
+    saturday: Optional[HvacDaySchedule]
+    sunday: Optional[HvacDaySchedule]
+
+    def for_json(self) -> Dict[str, Any]:
+        """Create dict for json."""
+        ret = {
+            "id": self.id,
+            "activated": self.activated,
+        }
+        for day in [
+            "monday",
+            "tuesday",
+            "wednesday",
+            "thursday",
+            "friday",
+            "saturday",
+            "sunday",
+        ]:
```

Comment:
This list should be shared with ChargeSchedule - maybe you can refactor ChargeSchedule at the same time ;)